openrpc/src/schemas/Event.json

```diff
@@ -35,6 +35,10 @@
             },
             "value": {
                 "$ref": "#/components/schemas/EventValue"
+            },
+            "txHash": {
+                "type": "string",
+                "description": "The transaction which triggered this event."
```

Comment:
is it worth adding schema definition for `TxHash` in https://github.com/stellar/stellar-docs/blob/addTxHash-getEvents/openrpc/src/schemas/Transaction.json, and `ref` it here?

Reply:
Great idea to `ref`! Done in ee4d825/

Reply:
It's fine in `Hash.json` imo :+1: just didn't see it